docker/omscore-nginx/nginx.conf

```diff
@@ -11,7 +11,7 @@ events {
 
 http {
   server_tokens off;
-  sendfile on;
+  sendfile off;
```

Comment:
Did you have a specific reason to turn this off?

Reply:
Yes it's creating troubles in docker containers and is merely a performance optimization of nginx